plugins/CuraEngineBackend/StartSliceJob.py

```diff
@@ -106,6 +106,10 @@ def _checkStackForErrors(self, stack: ContainerStack) -> bool:
         if stack is None:
             return False
 
+        # if there are no per-object settings we don't need to check the other settings here
+        if stack.getTop() == None or len(stack.getTop().getAllKeys()) == 0:
```

Comment:
```suggestion
        if stack.getTop() is None or not stack.getTop().getAllKeys():
```
Comparison with none should be done with is, and an an empty list also evaluates to false.